src/db/repository.py

```
@@ -0,0 +1,767 @@
+                "transcribed": sum(
+                    1 for e in episodes if e.transcript_status == "completed"
+                ),
+                "indexed": sum(
+                    1 for e in episodes if e.file_search_status == "indexed"
+                ),
+                "fully_processed": sum(1 for e in episodes if e.is_fully_processed),
+            }
+
+    def get_overall_stats(self) -> Dict[str, Any]:
+        """Get overall system statistics."""
+        with self._get_session() as session:
```

Ruff (0.14.7)

Local variable `session` is assigned to but never used

Remove assignment to unused variable `session`

(F841)